.github/workflows/build-and-test.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: |
           # notebooks should not contain outputs
           jupyter nbconvert --clear-output --inplace docs/examples/*.ipynb
-          if [[ `git status --porcelain` ]]; then
+          if [[ `git status --porcelain docs/examples` ]]; then
```

Comment:
I know it is reasonable to make this change, but could you please briefly illustrate how the previous implementation has caused failures in Github workflows?

Reply:
The Prometheus connector module uses a shade plugin to build the fat jar, producing a `dependency-reduced-pom.xml`. The `git status --porcelain` will complain about the new file `dependency-reduced-pom.xml`.